generators/2.0/java/templates/spring/build.gradle.ejs

```diff
@@ -0,0 +1,31 @@
+plugins {
```

Comment:
Not sure that we should redefine another `build.gradle.ejs` and `pom.xml.ejs` for Spring. Could you use [ejs](https://ejs.co/#docs) templating syntax with `if/else` within the same file instead?